docs/docs/articles/run-tests-with-github-actions.md

```diff
@@ -0,0 +1,143 @@
+# Run Tests with GitHub Actions
+
+**Run on Testkube** is a GitHub Action for running tests on the Testkube platform.
+
+Use it to run tests and test suites and obtain results directly in the GitHub's workflow.
+
+## Usage
+To use the action in your GitHub workflow, use kubeshop/testkube-run-action@v1 action. The configuration options are described in the Inputs section and may vary depending on the Testkube solution you are using (cloud or self-hosted) and your needs.
```

Comment:
1. Do we use inline code snippets, like `kubeshop/testkube-run-action@v1` instead of kubeshop/testkube-run-action@v1?
2. Do we use links for the internal sections (i.e. should "Inputs" here lead to the "Inputs" section below?)

Reply:
1. Yes, we should use  inline code snippet (achieve with two backticket ``) 
2. No

Reply:
Updated